static/js/modals/CreateSnapshotForm.tsx

```diff
@@ -79,28 +79,25 @@ const CreateSnapshotForm: FC<Props> = ({ instance, close, notify }) => {
   });
 
   return (
-    <Form onSubmit={formik.handleSubmit}>
-      <Modal
-        title={`Create snapshot for ${instance.name}`}
-        // TODO: put back the close prop on next react-components update
-        buttonRow={
-          <>
-            <Button
-              className="u-no-margin--bottom"
-              type="button"
-              onClick={close}
-            >
-              Cancel
-            </Button>
-            <SubmitButton
-              isSubmitting={formik.isSubmitting}
-              isDisabled={!formik.isValid}
-              buttonLabel="Create snapshot"
-            />
-          </>
-        }
-        onKeyDown={handleEscKey}
-      >
+    <Modal
+      title={`Create snapshot for ${instance.name}`}
+      // TODO: put back the close prop on next react-components update
+      buttonRow={
+        <>
+          <Button className="u-no-margin--bottom" type="button" onClick={close}>
+            Cancel
+          </Button>
+          <SubmitButton
+            isSubmitting={formik.isSubmitting}
+            isDisabled={!formik.isValid}
+            buttonLabel="Create snapshot"
+            onClick={() => void formik.submitForm}
```

Comment:
Why the `Modal` outside the `Form`? We could have avoided the redirection of the `click` event.

Reply:
It creates a visual bug, the CreateSnapshot Button goes into its own row when the modal is wrapped in a form.